src/plugins/workspace/public/components/add_collaborators_modal/add_collaborators_modal.test.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { AddCollaboratorsModal } from './add_collaborators_modal';
```

Comment:
Nit: keep a line break
```suggestion
import { AddCollaboratorsModal } from './add_collaborators_modal';

```